src/gamemodes.py

```diff
@@ -520,22 +521,23 @@ class AleatoireMode(GameMode):
     def __init__(self, arg=""):
         super().__init__(arg)
         self.SHARPSHOOTER_CHANCE = 1
-                                              #    SHAMAN   , CRAZED SHAMAN
-        self.TOTEM_CHANCES = {       "death": (      4      ,      1      ),
-                                "protection": (      8      ,      1      ),
-                                   "silence": (      2      ,      1      ),
-                                 "revealing": (      0      ,      1      ),
-                               "desperation": (      1      ,      1      ),
-                                "impatience": (      0      ,      1      ),
-                                  "pacifism": (      0      ,      1      ),
-                                 "influence": (      0      ,      1      ),
-                                "narcolepsy": (      0      ,      1      ),
-                                  "exchange": (      0      ,      1      ),
-                               "lycanthropy": (      0      ,      1      ),
-                                      "luck": (      0      ,      1      ),
-                                "pestilence": (      1      ,      1      ),
-                               "retribution": (      4      ,      1      ),
-                              "misdirection": (      0      ,      1      ),
+                                              #    SHAMAN   , CRAZED SHAMAN , WOLF SHAMAN
+        self.TOTEM_CHANCES = {       "death": (      4      ,       1       ,      0      ),
+                                "protection": (      8      ,       1       ,      0      ),
+                                   "silence": (      2      ,       1       ,      0      ),
+                                 "revealing": (      0      ,       1       ,      0      ),
+                               "desperation": (      1      ,       1       ,      0      ),
+                                "impatience": (      0      ,       1       ,      0      ),
+                                  "pacifism": (      0      ,       1       ,      0      ),
+                                 "influence": (      0      ,       1       ,      0      ),
+                                "narcolepsy": (      0      ,       1       ,      0      ),
+                                  "exchange": (      0      ,       1       ,      0      ),
+                               "lycanthropy": (      0      ,       1       ,      0      ),
+                                      "luck": (      0      ,       1       ,      0      ),
+                                "pestilence": (      1      ,       1       ,      0      ),
+                               "retribution": (      4      ,       1       ,      0      ),
+                              "misdirection": (      0      ,       1       ,      0      ),
+                                    "deceit": (      0      ,       1       ,      0      ),
```

Comment:
wolf shaman can't be all 0's here, needs to have something (possibly default, possibly tweaked). They can be introduced via amnesiac so having all 0's would horribly break that.